inc/agl.inc

```diff
@@ -17,6 +17,18 @@ XT_QUIRK_UNPACK_SRC_URI += " \
 # these layers will be added to bblayers.conf on do_configure
 XT_QUIRK_BB_ADD_LAYER += " \
     meta-xt-images-domx \
+    meta-virtualization \
+    meta-selinux \
+    meta-clang \
+"
+
+# Set default revision of AGL auxiliary layers
+BRANCH ?= "master"
```

Comment:
I would set it ti morty as it was before

Reply:
Done